lua/gk.lua

```diff
@@ -15,8 +15,39 @@ return function (net_conf, numa_table)
 	local ggu_lcore = table.remove(gk_lcores)
 	gatekeeper.gk_assign_lcores(gk_conf, gk_lcores)
 
+	gk_conf.max_num_ipv4_rules = 1024
+	gk_conf.num_ipv4_tbl8s = 256
+	gk_conf.max_num_ipv6_rules = 1024
+	gk_conf.num_ipv6_tbl8s = 65536
+
+	local segments = {
+		{
+			["ip_addr"] = "10.0.0.1",
+			["prefix_len"] = 24,
+			["policy_id"] = 0,
+		},
+	}
+
+	local policies = {
+		{
+			["policy_id"] = 0,
+			["action"] = gatekeeper.c.GK_FWD_GT,
+			["grantor_id"] = 0,
+		},
+	}
+
+	local grantors = {
+		"187.73.35.240",
+		"187.73.35.241",
+		"3ffe:2501:200:3::1",
+	}
+
+	local ret = gatekeeper.init_lpm(gk_conf,
+		net_conf, segments, policies, grantors)
+	if ret < 0 then return nil end
```

Comment:
What's the reason for separating `init_lpm()` (and `lua_init_gk_lpm()`) from the `run_gk()` function? Is it because `lua_init_gk_lpm()` requires so many parameters? Perhaps it is OK, but I it stood out to me because by making it a separate function, we have to be careful: for example, you made sure to set `gk_conf->net` in `lua_init_gk_lpm()` and took it out of `run_gk()`. However, you did not first make sure that the `net_conf` passed in as a parameter is not `NULL`, like we have in `run_gk()`.